contrib/create_yaml_files.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+import re
+import argparse
+from typing import Tuple
+
+CHANGE_FILE_NAME = True
+"""Whether to shorten file names by removing the verdicts, if possible without collisions."""
+AVOID_COLLISIONS_ACROSS_DIRECTORIES = True
```

Comment:
These two look like they should be arguments instead of constants?

Reply:
I felt that they should be changed that infrequently that I didn't make the effort to transform them, but it's done now.